polyfill.js

```diff
@@ -59,15 +59,46 @@
     }
 
     /**
-     * @param {TypedArray} species
+     * @param {TypedArray} arr
+     * @returns {TypedArray[typeof Symbol.toStringTag]}
+     */
+    function typedArrayNameInternalSlot(arr) {
+        return Object.getOwnPropertyDescriptor(typedArrayPrototype, Symbol.toStringTag)
+            .get.call(arr);
+    }
+
+    /**
+     * @param {TypedArray} example
      * @param {number} length
      * @returns {TypedArray}
      */
-    function typedArraySpeciesCreate(species, length) {

```

Comment:
Oh, this definition was really incorrect, but I guess you realize that now...

Reply:
😅